frontend/src/pages/ConnectedPages/Game/Countdown/index.tsx

```diff
@@ -18,7 +24,7 @@ const Countdown: React.FC<ICountdown> = ({ setStep }: ICountdown) => {
         setStep(IGameEnum.QUIZZ);
       }
     }, 1000);
-  }, [countdown]);
+  }, [countdown, setStep]);
```

Comment:
I know there is a warning but I do not like seen the setStep here

Reply:
I don't understand ^^